spring-cloud-gcp-starters/spring-cloud-gcp-starter-core/src/main/java/org/springframework/cloud/gcp/core/autoconfig/GoogleCloudContextAutoConfiguration.java

```diff
@@ -14,26 +17,26 @@
 
 /**
  * @author Vinicius Carvalho
+ *
+ * Base starter for Google Cloud Projects. Provide defaults for {@link GoogleCredentials}.
+ * Binds properties from {@link org.springframework.cloud.gcp.core.GoogleCloudProperties}
+ *
  */
 @Configuration
 @ConditionalOnClass(GoogleCredentials.class)
-public class CredentialsAutoConfiguration implements EnvironmentAware {
+@EnableConfigurationProperties(GoogleCloudProperties.class)
+public class GoogleCloudContextAutoConfiguration {
```

Comment:
It's Spring Boot stuff. SI is far away from there.

Well, auto-config means provisioning of some beans into the application context and it make it based on some properties. We really don't configure here some `Properties` bean. 
I guess we are going to provide some common beans in this auto-config which really are related to the GCP Context. 
I don't have particular preferences to the name, but seems for me the whole name sounds much official than abbreviation.